packages/cra-template-typescript/template/src/index.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
-import * as serviceWorker from './serviceWorker';
+import * as serviceWorkerRegistration from './serviceWorkerRegistration';
```

Comment:
Here's another instance of the renaming. Not sure if this needs to be automated or what.

Reply:
We're okay calling this out in release notes of `4.0`. We can provide a recommendation for how to transition over to this new strategy.